src/data/zha.ts

```diff
@@ -73,7 +73,7 @@ export interface ClusterAttributeData {
 export interface AttributeConfigurationStatus {
   id: number;
   name: string;
-  success: boolean | undefined;
+  status: string;
```

Comment:
```suggestion
  status: "SUCCES" | "FAILURE" | "UNSUPPORTED_ATTRIBUTE" | "UNREPORTABLE_ATTRIBUTE";
```

Reply:
Or can it be any of these?

https://github.com/zigpy/zigpy/blob/dev/zigpy/zcl/foundation.py#L22

Reply:
I need to go through them to see which ones could possibly be returned. In this context we only care about a few.